src/pages/index.tsx

```diff
@@ -53,11 +53,67 @@ export async function getStaticProps(
   }
 }
 
+const sections = [
+  {
+    title: 'Indonesia Starterpack',
+    description: <p>Situs atau service ini terpopuler dengan orang Indonesia</p>,
+    sites: idnStarterPack
+  },
+  {
+    title: 'Essential Developer Toolkit',
+    description: <p>Service yang ✨ anak bangsa ✨ mungkin akan pakai saat membuat app</p>,
+    sites: devStarterPack
+  },
+  {
+    title: 'Top websites USA',
+    description: (
+      <>
+        <p>
+          2022 Top websites in the USA. Sumber data website diambil dari{' '}
+          <a
+            href='https://www.semrush.com/blog/most-visited-websites/'
+            className='text-blue-700 underline'
+            target='_blank'
+            rel='noreferrer'
+          >
+            semrush.com
+          </a>
+        </p>
+        <p>
+          <em>Catatan: Ada beberapa website ditiadakan karena tidak berhubungan</em>
+        </p>
+      </>
+    ),
+    sites: websiteListUSA
+  },
+  {
+    title: 'Linux Starterpack',
+    description: <p>Service yang sering dipakai Linux user 🐧</p>,
+    sites: linuxStarterPack
+  }
+]
+
 const IndexPage = ({ PSEData: data, blockData }: IndexPageProps) => {
+  const [trustPositifMap, setTrustPositifMap] = React.useState<Map<String, String>>(new Map())
+
+  React.useEffect(() => {
+    fetch('/api/trustPositif', {
+      method: 'POST',
+      body: new URLSearchParams({
+        name: allWebsitesCombined
+          .map((item) => item.website.replace(/(https?:\/\/)/, '').replace(/www\./, ''))
+          .join('\n')
+      })
+    })
+      .then((res) => res.json())
+      .then((res) =>
+        setTrustPositifMap(new Map(res.values.map((item: any) => [item.Domain, item.Status])))
+      )
+  }, [])
+
```

Comment:
It would be great if we fetch TrustPositif's data in NextJS' `getStaticProps` so just to reduce the network overhead that the gov'd website will get.

For context, today alone we are handling ~120k unique visits. I don't want to accidentally DDOS TrustPositif's service 😅

Reply:
Hmm, if you want i think i can try creating a workflow that'll update it every n hours just like how the site currently does with indi.wtf, i'll look into it after finishing up some work

Reply:
Would be much better if you could make use of their multiple query ability (with 100 TLD query max per request), that could significantly reduce network overhead to the gov's site. (everything could be done in 3 requests per deploy)

Reply:
> Would be much better if you could make use of their multiple query ability (with 100 TLD query max per request), that could significantly reduce network overhead to the gov's site. (everything could be done in 3 requests per deploy)
> 
> EDIT: seems like you're throwing every single entry at once, please split to 100 sites to respect their limit

Implemented, thanks for the suggestion

Reply:
> It would be great if we fetch TrustPositif's data in NextJS' `getStaticProps` so just to reduce the network overhead that the gov'd website will get.
> 
> For context, today alone we are handling ~120k unique visits. I don't want to accidentally DDOS TrustPositif's service sweat_smile

Implemented, wdyt?

Reply:
RSLGTM